extensions/transmission/src/utils/renderDetails.ts

```diff
@@ -0,0 +1,68 @@
+import dedent from "dedent-js";
+import prettyBytes from "pretty-bytes";
+import type { Torrent } from "../listTorrents";
+import { formatDate } from "./date";
+import { renderPieces } from "./renderCells";
+
+const NL = "  \n";
+
+export const renderDetails = (torrent: Torrent, downloadStats: string) => {
+  const img = renderPieces(torrent.pieces, torrent.pieceCount);
+
+  return dedent(`
```

Comment:
TBH it'd would be great if there was a better way to render this data. Maybe some ad-hoc components that can be used into a Detail view to format text instead of markdown?

Reply:
Interesting idea, probably we can somehow make this easier. Could you describe this as an [GH Issue](https://github.com/raycast/extensions/issues/new?assignees=&labels=api%2C+feature+request&template=api_feature_request.md&title=) so we can keep track on this?